packages/smooth_app/lib/pages/product/new_product_page.dart

```diff
@@ -219,33 +226,133 @@ class _ProductPageState extends State<NewProductPage> {
           top: 4.0,
           bottom: 20.0,
         ),
-        insets: const EdgeInsets.all(12.0),
-        child: Column(children: <Widget>[
-          Align(
+        insets: EdgeInsets.zero, // Zero padding for the card content.
+        clipBehavior: Clip.antiAliasWithSaveLayer,
```

Comment:
thanks for catching, my plan was to change this to 'antiAlias' but got slipped through the cracks.

Why we need it ?:
To be honest, I witnessed a strange behavior in the Card which I can't explain.with Clip.none & with "insets (which is padding inside the container within the card) = 0" the card doesn't have rounded corners anymore, I don't fully understand why. 
Using Clip.antiAlias fixes the issue, in fact I think we can even make Clip.antiAlias the default behavior for SmoothCard, as that seems to be what we want.

Let me know what you think.

Reply:
Actually, looking at the docs for `Clip.none` (cited below), it looks like clipping only matters when the content overflows the widget boundary. I don't see why we should rely on overflows in this widget. Should we try to refactor the code to avoid the overflows or at least document our shortcut approach here?
 
> This is the default option for most widgets: if the content does not overflow the widget boundary, don't pay any performance cost for clipping.

Reply:
mmm not sure I understand why there's an overflow, I thought it may be because of the `Padding` at first but it's not even that. I added a comment for now, but let me take a stab at refactoring the `SmoothCard` class in a follow up PR (with better parameter naming), i'll try to fix it if it's not too complicated.